### PR TITLE
feat(snippet-bot): allow api_shortname for samplegen samples

### PR DESCRIPTION
**Pull request by tmatsuo**
With this PR, snippet-bot will start allowing api_shortname as the
prefix for region tags generated by samplegen.

### CODE REVIEW

File: packages/snippet-bot/test/violations.ts [hunk 1]
```diff
@@ -25,13 +29,74 @@ import {
...
+import {ApiLabels} from '../src/api-labels';
+import * as apiLabelsModule from '../src/api-labels';
 import {resolve} from 'path';
 import assert from 'assert';
 import {describe, it} from 'mocha';
 import * as sinon from 'sinon';
 
 const fixturesPath = resolve(__dirname, '../../test/fixtures');
 
+describe('checkProductPrefixViolations', async () => {
```
Comment (bcoe) [inline packages/snippet-bot/test/violations.ts:41]: no need for `async` here 👍 I think you can use `async`, if you do some async initialization:

```
desceribe('checkProductPrefixViolations', async () => {
  const sharedConfig = await loadConfig();
});
```

But I'd only do this if needed.

Comment (bcoe) [reply packages/snippet-bot/test/violations.ts:41]:
```suggestion
describe('checkProductPrefixViolations', () => {
```

**Review by bcoe — CHANGES_REQUESTED** [review]
One small nit, LGTM otherwise.

File: packages/snippet-bot/test/violations.ts (hunk 1 above)
Comment (tmatsuo) [reply packages/snippet-bot/test/violations.ts]: Sure, removed!

**Comment by tmatsuo** [comment]
Sorry, I'll create a PR from the same repo, not from my fork

**Pull request closed without merging**